_docs/issuetracking.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Issue Tracking
+subtitle: This section will discuss how to use an issue tracker with GitLive
```

Comment:
change that subtitle to something more fitting. we never started a section with "This section"